src/app/layouts/base-layout/components/header/header.tsx

```diff
@@ -29,12 +29,27 @@ const navLinks = [
     href: ROUTES.COMMUNITY,
     dropdownInner: <SchoolMenu heading="community" color="dark" hasTitle={false} />,
   },
+  {
+    label: 'Mentorship',
+    href: ROUTES.MENTORSHIP,
+    dropdownInner: <SchoolMenu heading="mentorship" color="dark" hasTitle={false} />,
+  },
 ];
 
 export const Header = () => {
   const [isMenuOpen, setMenuOpen] = useState(false);
-  const [color, setColor] = useState('gray');
   const { key, hash, pathname } = useLocation();
+
+  useEffect(() => {
+    if (pathname.includes(ROUTES.MENTORSHIP)) {
+      setColorName('blue');
+    } else {
+      setColorName('gray');
+    }
+  }, [pathname]);
```

Comment:
Please move this below the declaration of the other hooks

Reply:
Done
Would you please explain what this was required for? The style of the code?

Reply:
> Done Would you please explain what this was required for? The style of the code?

Yeah, just for the code consistency